front/lib/actions/mcp_actions.ts

```diff
@@ -144,10 +144,15 @@ export function getToolExtraFields(
       return r;
     }
     const serverName = r.value.name;
-    // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
-    toolsStakes = INTERNAL_MCP_SERVERS[serverName].tools_stakes || {};
+    toolsStakes = {
+      ...(INTERNAL_MCP_SERVERS[serverName].tools_stakes ?? {}),
+    };
```

Comment:
why the deconstruction and reconstruction ? can't we just "?? {}" and not "{...([].tools_stakes ?? {})}"